packages/plugin-ext/src/main/browser/documents-main.ts

```diff
@@ -42,7 +42,7 @@ export class DocumentsMainImpl implements DocumentsMain {
         modelService: EditorModelService,
         rpc: RPCProtocol,
         private editorManger: EditorManager,
-        private openerService: OpenerService
+        private openerService: OpenerService,
```

Comment:
final commas are allowed but  I think we shouldn't add it in existing code
WDYT?

Reply:
OK

Reply:
it avoids to edit the previous lines when adding new code. For example the previous line about editormanager was not modified when adding this line so to me I find it very nice.